src/Components/CommentsWidget.js

```diff
@@ -77,16 +77,50 @@ export class Comment extends preact.Component {
                     []
                 )}
                 <p>
-                    {this.props.message.split('\n').map(item => (
-                        <span>
-                            {item}
-                            <br />
-                        </span>
-                    ))}
+                    {this.props.message
+                        .split('\n')
+                        .map(this.processLine)
+                        .flat()}
                 </p>
             </div>
         );
     }
+
+    /**
+     * Handle some very basic markup in comments, similar to what the generator allows.
+     *
+     * The following tags are supported:
+     *   - <bold>This text will be bold.</bold>
+     *   - <italic>This text will be italic.</italic>
+     *   - <link url="https://example.org">This text will link to example.org.</link> Note that the URL can technically
+     *       be missing, it will then produce a link without href. Also note that URLs are not checked at all.
+     *
+     * @param {string} line The line of text to process.
+     * @returns {Array} An array of the elements to insert instead of the line.
+     */
+    processLine = line => {
+        const processChunk = function(chunk) {
+            let chunks = [];
+            let remaining = chunk;
+            let match;
+
+            while (
+                (match = remaining.match(/<(?<tag>bold|italic|link)(?: url="(?<url>.+?)")?>(?<content>.+?)<\/\1>/))
```

Comment:
> This wouldn't match `<link url=""></link>`.

That is definitely on purpose. I don't see a reason to match an empty link. 

> And if yes, why not require the url to actually be a (https) url?

Sure, we can do that.

Reply:
> Empty tags are also not matched. Why is that? Currently, emtpy tags will just remain in the comment.

Is that a problem? These tags aren't really meant for 'public consumption'. They are mostly for us to actually be able to have proper links in comments. (The formatting stuff is just there because it was too easy. :D) I don't mind other people using them, but they are not the target 'audience' (for lack of a better term).

As such, I don't really see a reason to handle empty tags. That's just not a valid construct in this PMML (pseudo mini markup language—when do I get my Wikipedia page? :D) and thus remains plain text. \*shrug*

Reply:
I guess datenanfragen.de could actually get a wikipedia page someday…

For everything else: Sure. We might get some ugly looking comments that way but I dont care.

Reply:
```suggestion
                (match = remaining.match(/<(?<tag>bold|italic|link)(?: url="(?<url>https:\/\/.+?)")?>(?<content>.+?)<\/\1>/))
```